osu.Game.Rulesets.Taiko/UI/TaikoPlayfield.cs

```diff
@@ -226,20 +210,12 @@ private void addDrumRollHit(DrawableDrumRollTick drawableTick)
         {
             bool isStrong = drawableTick.HitObject.IsStrong;
             double time = drawableTick.HitObject.GetEndTime();
-
-            DrawableHit drawableHit;
-            if (drawableTick.JudgementType == HitType.Rim)
-                drawableHit = new DrawableFlyingRimHit(time, isStrong);
-            else
-                drawableHit = new DrawableFlyingCentreHit(time, isStrong);
-
-            drumRollHitContainer.Add(drawableHit);
+            drumRollHitContainer.Add(new DrawableFlyingHit(time, isStrong));
```

Comment:
This needs to have a type attached to it. I believe all of these will appear as centre hits right now?